2020/day5/solution.py

```diff
@@ -0,0 +1,41 @@
+import itertools
+
+class Ticket:
+    _row = None
+    _column = None
+
+    def __init__(self, code):
+        self._code = code
+    
+    @property
+    def seat_id(self):
+        return self.row * 8  + self.column
+    
+    @property
+    def row(self):
+        if self._row == None:
+            self._row = int(self._code[:7].replace("F","0").replace("B","1"), base=2)
+        return self._row
+
+    @property
+    def column(self):
+        if self._column == None:
+            self._column = int(self._code[-3:].replace("L","0").replace("R", "1"), base=2)
+        return self._column
+    
+with open('input.txt') as f:
+    tickets = [Ticket(line.strip()) for line in f]
+    max_id = max(ticket.seat_id for ticket in tickets)
+    print("Max Seat ID for flight is :", max_id)
```

Comment:
```suggestion
    print(f"Max Seat ID for flight is: {max_id}")
```

Reply:
sweet.   c# had something similar.  TIL that python allows variable interpolation with f-strings